tests/framework/CodeInterfaceTests/test_Prescient_code_interface_fake.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" ?>
+<Simulation>
+  <TestInfo>
+    <name>framework/CodeInterfaceTests/PrescientInterface</name>
+    <author>@cogljj</author>
+    <created>2020-09-02</created>
+    <revisions>
+    </revisions>
+    <classesTested>CodeInterfaces.PrescientCodeInterface</classesTested>
+    <description>
+      Test the Prescient Code Interface.
```

Comment:
we need a expanded description here... please write what we test, the variables we are perturbing, the expected output etc.

Reply:
Fixed.